webhook-service/handler/handler.go

```diff
@@ -84,7 +85,7 @@ func (th *TaskHandler) Execute(keptnHandler sdk.IKeptn, event sdk.KeptnEvent) (i
 		}
 		err = keptnHandler.SendFinishedEvent(event, result)
 		if err != nil {
-			return nil, sdkError(fmt.Sprintf("could not send finished event: %s", err.Error()), err)
+			return nil, sdkError(fmt.Sprintf("could not send finished event: request '%s' %s", webhook.Requests[0], err.Error()), err)
```

Comment:
Can it happen that `webhook.Requests` is empty at this point? Also, is the first of the webhook requests actually tied to this particular error (sending the finished event failed)?

Reply:
Actually it shouldn't happen, we are now checking that webhook.Requests are not empty, see this PR below

Reply:
ok thank you for clarifying :) But still, adding the first of the requests to this error message might be misleading, as the error in this case does not see, to be tied to that particular request